plugins/editor/js/editor.js

```diff
@@ -47,6 +39,9 @@
             stylesheetsInclude.push(editorWysiwygCSS + '?v=' + editorVersion);
         }
 
+        // If you want to include a custom stylesheet inside the iFrame of the wysiwyg
+        stylesheetsInclude.push(gdn.meta.currentThemePath + '/design/editorWysiwyg.css?v=' + editorVersion);
```

Comment:
```suggestion
        var currentThemePath = gdn.definition("currentThemePath", null);
        if (currentThemePath) {
            stylesheetsInclude.push(currentThemePath + "/design/editorWysiwyg.css?v=" + editorVersion);
        }
```

Even still, isn't this just gonna throw a 404 in the background and cause a pointless request to the server if the theme doesn't have an editorWysiwyg.css?